src/App.jsx

```diff
@@ -1,40 +1,44 @@
-import React from 'react';
-import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import 'bulma/css/bulma.css';
+import { useState } from 'react';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+export const App = () => {
+  const getCurrentTab = (tabID) => {
+    const newTab = tabs.find(tab => tab.id === tabID);
+
+    if (!newTab) {
+      setSelectedTabId(tabs[0].id);
+
+      return getCurrentTab(tabs[0].id);
+    }
+
+    return newTab;
+  };
+
+  const setNewTab = (tab) => {
+    setSelectedTabId(tab.id);
+  };
+
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const currentTab = getCurrentTab(selectedTabId);
+
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${currentTab.title}`}</h1>
```

Comment:
so in short `<h1 className="title">Selected tab is {currentTab.title}</h1>`
if the project swears at this format, then please let me know about it

Reply:
Kinda
![image](https://github.com/mate-academy/react_tabs-js/assets/55620816/90c9031d-05ab-4ac7-9b8f-93e12e53bdda)